core/workspace_svg.js

```diff
@@ -247,6 +249,14 @@ Blockly.WorkspaceSvg.prototype.useWorkspaceDragSurface_ = false;
  */
 Blockly.WorkspaceSvg.prototype.isDragSurfaceActive_ = false;
 
+/**
```

Comment:
Moved up a little bit to match where it is in blockly/get rid of a merge conflict.  I don't remember whether it's used in this change.